integration-tests/storage_test.go

```diff
@@ -17,32 +20,83 @@ var _ = Describe("Storage Bucket", func() {
 		Expect(mockTerraform.Reset()).To(Succeed())
 	})
 
+	It("should publish in the catalog", func() {
+		catalog, err := broker.Catalog()
+		Expect(err).NotTo(HaveOccurred())
+
+		service := testframework.FindService(catalog, serviceName)
+		Expect(service.ID).NotTo(BeNil())
```

Comment:
nitpick: I think that `service.ID` is a string type in Go, which can never be `nil`. A better test would be to use the `Empty()` matcher, or even better, match against the value expected. Same applies for `ImageUrl`, `DisplayName`, `Name`, etc...